posthog/management/commands/start_temporal_worker.py

```diff
@@ -70,6 +76,7 @@
     GENERAL_PURPOSE_TASK_QUEUE: PROXY_SERVICE_ACTIVITIES
```

Comment:
Linking everything to general-purpose task queue (as other AI workflows), could justify creating a new queue when the scale grows.